apps/patient/src/utils/text.ts

```diff
@@ -126,13 +126,18 @@ export const text = {
   tracking: 'Tracking #:',
   tryPhoton: 'Try Photon',
   urgencyOptions: [
-    '10:00 am',
-    '12:00 pm',
-    '2:00 pm',
-    '4:00 pm',
-    '6:00 pm',
-    'After hours',
-    'Tomorrow'
+    { label: '10:00 am', description: null, icon: false },
```

Comment:
Kind of lame that i changed all of this to an object because of just the urgent one, but I like this more than conditionally showing in the Urgency view

Reply:
you could do  `{ label: '10:00 am' }` since you check for description below

also could enforce options as `{ label: string; description: string; icon: boolean }  | { label: string };`